File: iOS/DoraemonKitDemo/DoraemonKitDemo/Plugin/TestPlugin.m
//
//  TestPlugin.m
//  DoraemonKitDemo
//
//  Created by yixiang on 2017/12/11.
//  Copyright © 2017年 yixiang. All rights reserved.
//

#import "TestPlugin.h"


@implementation TestPlugin

- (void)pluginDidLoad{
    NSLog(@"TestPlugin pluginDidLoad");
}

- (void)pluginDidLoad:(NSDictionary *)itemData{
    NSLog(@"TestPlugin pluginDidLoad:itemData = %@",itemData);
}

@end

File: iOS/Swift/DoKitSwiftDemo/DoKitSwiftDemo/Plugin/TestPlugin.swift
//
//  TestPlugin.swift
//  DoKitSwiftDemo
//
//  Created by didi on 2020/5/13.
//  Copyright © 2020 didi. All rights reserved.
//

import UIKit
import DoraemonKit

@objc(TestPlugin)
class TestPlugin: NSObject,DoraemonPluginProtocol {
    @objc func pluginDidLoad(){
        print("pluginDidLoad")
    }
}
